aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -507,6 +514,24 @@ Resources:
     UpdateReplacePolicy: Delete
     DeletionPolicy: Delete
 
+  MarketingStaticAssetsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "cdo-marketing-static-assets-${Environment}"
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      VersioningConfiguration:
+        Status: Enabled
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpireOldVersions
+            Status: Enabled
+            NoncurrentVersionExpiration:
+              NoncurrentDays: 90
+
```

Comment:
I think this is enabled by default on all new buckets, but worth being explicit about server-side encryption:

```yaml
      BucketEncryption:
        ServerSideEncryptionConfiguration:
          - ServerSideEncryptionByDefault:
              SSEAlgorithm: "AES256"      BucketEncryption:
        ServerSideEncryptionConfiguration:
          - ServerSideEncryptionByDefault:
              SSEAlgorithm: "AES256"
```

Reply:
Updated to include. I did confirm it seems to be default enabled.